packages/microservices/interfaces/microservice-configuration.interface.ts

```diff
@@ -78,6 +79,7 @@ export interface TcpOptions {
     retryAttempts?: number;
     retryDelay?: number;
     serializer?: Serializer;
+    tlsOptions?: TlsOptions;
```

Comment:
And you are using ``TlsOptions`` here

```suggestion
    tlsOptions?: ConnectionOptions;
```